webpack/assets/javascripts/react_app/components/notifications/index.js

```diff
@@ -8,9 +8,9 @@ import { groupBy, some, isUndefined } from 'lodash';
 
 class notificationContainer extends React.Component {
   componentDidMount() {
-    const { getNotifications, data: { url } } = this.props;
+    const { startNotificationsPolling, data: { url } } = this.props;
 
-    getNotifications(url);
+    startNotificationsPolling(url);
```

Comment:
another question: would it be better to change the react mounter to not mount if the div is already mounted? or in other words, not allow turbolinks or some other ajax call to remount an already mounted component? this way we will not need to face any of these challenges?